include/ignition/gazebo/components/LinearVelocityCmd.hh

```diff
@@ -32,15 +32,17 @@ namespace gazebo
 inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
 namespace components
 {
-  /// \brief A component type that contains linear velocity of an entity
-  /// represented by ignition::math::Vector3d.
+  /// \brief A component type that contains commandad linear velocity of an
```

Comment:
I'm not sure what `commandad` is supposed to mean here, same for below comment

Reply:
It's when your dad commands you to do something :upside_down_face: